src/genezio.ts

```diff
@@ -179,6 +179,7 @@ program
     .command("deploy")
     .option("--backend", "Deploy only the backend application.", false)
     .option("--frontend", "Deploy only the frontend application.", false)
+    .option("--name <name>", "Name of the component to deploy.")
```

Comment:
```suggestion
    .option("--name <name>", "Specify the name of the frontend to be deployed .")
```

Reply:
This option doesn't make sense for deploying backend, ssr or containers. We should mildly specify that so the user won't be confused.